src/components/salesEvidenceSearch.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ChangeEvent } from 'react'
+import { IntersectedSearchAndFilterParams } from '../types'
+import { SearchWidget } from './SearchWidget'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+import { SaleTypeDropdown } from './SaleTypeDropdown'
+
+type SearchWidgetProps = {
+    searchParameters: IntersectedSearchAndFilterParams
+    changeSearchParameters: (newParameters: IntersectedSearchAndFilterParams) => void
+}
+
+export function SalesEvidenceSearchWidget (props: SearchWidgetProps) {
+    return (
+        <>
+            <SearchWidget changeSearchParameters={props.changeSearchParameters} searchParameters={props.searchParameters} />
+            <form className="wrapper">
+                <label className="eight">Land Area m2<br />
```

Comment:
2 should be superscript. You can use the <sup> tag

Reply:
Fixed